portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -37,11 +37,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     BlobstoreService blobstoreService =
         BlobstoreServiceFactory.getBlobstoreService();
     String uploadUrl = blobstoreService.createUploadUrl("/data");
+    System.out.println("The upload url is " + uploadUrl);
 
     response.setContentType("text/html");
     response.addHeader("Cache-Control",
                        "no-cache, no-store, must-revalidate, max-age=0");
-   /* response.addHeader("Expires", "0");*/
+    /* response.addHeader("Expires", "0");*/
```

Comment:
Remove the unused code. 
I think this will be fixed after merging with the previous pr.